### DISCUSSION TITLE
Yard sign generator

### DISCUSSION THREAD

**Discussion by mellobacon** [TehGM/Randominator#10] [Ideas]
Something like this: https://zachmu.github.io/yardsign/

But random. Could maybe have each line be different topics? Weird opinions about everyday but unpopular objects/concepts maybe?
"{thing(s)} are {enter opinion here}"

Idk. Im thinking something like madlibs  would be cool.

**Comment by TehGM** [comment]
Sounds good. I don't think I'll do it for v0.3 (I want to get it out soon, it's been dragging on for a while), but I'll probably schedule it for release v0.4.
